client/landing/stepper/declarative-flow/internals/steps-repository/bundle-confirm/index.tsx

```diff
@@ -252,7 +255,13 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 						<StyledNextButton
 							disabled={ isTransferringBlocked || ! isDataReady }
 							onClick={ () => {
-								recordTracksEvent( 'calypso_woocommerce_dashboard_confirm_submit', {
+								let eventName = 'calypso_bundle_dashboard_confirm_submit';
+
+								if ( 'woo-on-plans' === pluginSlug ) {
```

Comment:
This logic was applied in some different places as part of this commit: https://github.com/Automattic/wp-calypso/pull/84886/commits/4d3209580dd55d0173de427456027482704f11f9

Do you think it makes sense? So we keep the same event and log when it's the `woo-on-plans`, but we use a generic name for the others. Or maybe should we register one event and log for `woo-on-plans`, and always register the generic one (having 2 registers for `woo-on-plans`)?

Personally, I thought the ideal would be to just rename it for every case (the `calypso_woocommerce_dashboard_confirm_submit` wouldn't exist anymore), but I'm afraid that it could break existing reports, and so on.

Also, should I document this somewhere?

Reply:
Done in: https://github.com/Automattic/wp-calypso/pull/84886/commits/59878f27314cabf956c55e618cf751429dfaf91c. I wasn't very sure about the logstash, so I did the same there, just in case.

I'll take a look about the documentation! Thank you!

Reply:
Events are also registered. I'm just waiting for the checks of one of them to merge.
The previous ones weren't registered, and I didn't register them because we probably don't want them to be used.

https://github.com/Automattic/tracks-events-registration/pull/2054
https://github.com/Automattic/tracks-events-registration/pull/2055